tests/sagemaker/test_deployment.py

```diff
@@ -229,7 +229,7 @@ def test_deploy_creates_sagemaker_and_s3_resources_with_expected_names_and_env_f
         proxy_variables = {
             "http_proxy": "http://proxy.example.net:1234",
             "https_proxy": "https://proxy.example.net:1234",
-            "no_proxy": "localhost",
```

Comment:
Let's comment this is aws IP and change it as a global variable definition.

Reply:
Sure, so defining the AWS_IP as a variable `AWS_IP = "169.254.169.254"` over here https://github.com/mlflow/mlflow/blob/master/tests/helper_functions.py#L32 and then importing it over in `tests/sagemaker/test_deployment.py`, should be okay? Or did you have anything else in mind?

Reply:
sounds good.

Reply:
Can we replace 169.254.169.254 with a hostname ? And explain why we need to add no-proxy for it here ?

Reply:
It appears the IP `169.254.169.254 ` does not have a hostname but needs to be specified in `no_proxy` for interacting with AWS services.

Ref: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-proxy.html#cli-configure-proxy-ec2